functions/__sf_section_rust.fish

```diff
@@ -32,6 +33,10 @@ function __sf_section_rust -d "Display the current Rust version"
 
 	set -l rust_version (rustc --version | cut -d' ' -f2)
```

Comment:
Thinking it over, I think the "fish way" of doing this would be.

```fish
set -l rust_version (rustc --version | string split ' ')[2]
```

It's effectively the same, but would be more approachable for those familiar with the fish library. Thoughts?